cmd/update/project.go

```diff
@@ -5,36 +5,30 @@ import (
 	"fmt"
 
 	"github.com/flyteorg/flytectl/clierrors"
+	"github.com/flyteorg/flytectl/cmd/config/subcommand/project"
+	"gopkg.in/yaml.v2"
+
+	"io/ioutil"
 
-	"github.com/flyteorg/flytectl/cmd/config"
 	cmdCore "github.com/flyteorg/flytectl/cmd/core"
 	"github.com/flyteorg/flyteidl/gen/pb-go/flyteidl/admin"
 	"github.com/flyteorg/flytestdlib/logger"
 )
 
-//go:generate pflags ProjectConfig --default-var DefaultProjectConfig --bind-default-var
-
-// Config hold configuration for project update flags.
-type ProjectConfig struct {
-	ActivateProject bool `json:"activateProject" pflag:",Activates the project specified as argument."`
-	ArchiveProject  bool `json:"archiveProject" pflag:",Archives the project specified as argument."`
-	DryRun          bool `json:"dryRun" pflag:",execute command without making any modifications."`
-}
-
 const (
 	projectShort = "Update project resources"
 	projectLong  = `
 Updates the project according to the flags passed. Allows you to archive or activate a project.
 Activate project flytesnacks:
 ::
 
- flytectl update project -p flytesnacks --activateProject
+ flytectl update project --id flytesnacks --activateProject
 
```

Comment:
This breaks the older functionality. We should keep this backward compatible since incase this is used by teams in there automation already as it would break for them after an upgrade

Reply:
reverted the change